filetracker/__init__.py

```diff
@@ -856,5 +876,7 @@ def delete_file(self, name):
     def list_local_files(self):
         result = []
         if self.local_store:
-            result.extend(self.local_store.list_files())
+            for entry in self.local_store.list_files():
+                entry = entry._replace(client=self)
```

Comment:
Też mnie zaskoczyła nazwa rozpoczynająca się od '_', ale to jest w dokumentacji: https://docs.python.org/2/library/collections.html#collections.somenamedtuple._replace

Ponadto gdzieś indziej na stronie:
> In addition to the methods inherited from tuples, named tuples support three additional methods and two attributes. To prevent conflicts with field names, the method and attribute names start with an underscore.